test.sh

```diff
@@ -10,7 +10,7 @@ fi
 # Order doesn't matter. Note: godep-restore is specifically left out of the
 # defaults, because we don't want to run it locally (would be too disruptive to
 # GOPATH).
-RUN=${RUN:-vet fmt migrations unit integration errcheck}
+RUN=${RUN:-vet fmt migrations unit coverage integration errcheck}
```

Comment:
It seems like after this change, `docker-compose run boulder ./test.sh` on non-CI runs takes much longer due than before. Before it finished in a bearable amount of time but now it takes too long. I know there are more selective ways of running the tests, but was it really intentional to enable coverage by default for non-CI builds? If so it may be worth updating the documentation about how to run the tests to specifically mention how to work around this.